.github/autobuild/mac.sh

```diff
@@ -26,7 +26,7 @@
 
 set -eu
 
-QT_DIR=/usr/local/opt/qt
+QT_DIR=/opt/qt
```

Comment:
To fit to modern apple silicon maps similar to https://docs.brew.sh/Installation